src/components/ContentNav/SubItem/index.spec.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { SubItem } from '.';
+
+describe('SubItem', () => {
+  it('should be active', () => {
+    const subItemActive = mount(<SubItem name={'Test Name'} active={true} />);
+
+    subItemActive.simulate('click');
+    expect(subItemActive).toMatchSnapshot();
```

Comment:
We actually don't want to save the entire component as a snapshot; we want to render the component to JSON first, and then save that as a snapshot. Use the `toJson` helper in the `enzyme-to-json` package to accomplish this. Looking at the generated snapshot for the test should tell you why we would want to do this.

Reply:
Here's a link to the package: https://github.com/adriantoine/enzyme-to-json